pkg/helm/handlers/handlers.go

```diff
@@ -143,3 +149,70 @@ func (h *helmHandlers) HandleChartGet(user *auth.User, w http.ResponseWriter, r
 	res, _ := json.Marshal(resp)
 	w.Write(res)
 }
+
+
+func (h *helmHandlers) HandleHelmUpgradeRelease(user *auth.User, w http.ResponseWriter, r *http.Request) {
```

Comment:
No need to have `Helm` inside the function name - it is a part of helm handler package anyway.